Tests/HederaE2ETests/Schedule/ScheduleSign.swift

```diff
@@ -0,0 +1,162 @@
+/*
+ * ‌
+ * Hedera Swift SDK
+ * ​
+ * Copyright (C) 2022 - 2023 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permiss  ions and
+ * limitations under the License.
+ * ‍
+ */
+
+import Hedera
+import XCTest
+
+internal final class ScheduleSign: XCTestCase {
+
+    internal func testTransferSign() async throws {
+        let testEnv = try TestEnvironment.nonFree
+
+        let key1 = PrivateKey.generateEd25519()
+        let key2 = PrivateKey.generateEd25519()
+
+        let key_list: KeyList = [.single(key1.publicKey), .single(key2.publicKey)]
```

Comment:
```suggestion
        let keyList: KeyList = [.single(key1.publicKey), .single(key2.publicKey)]
```
(don't accept the change as-is, you'll need to change other lines too